table/block_based/filter_policy.cc

```diff
@@ -610,12 +625,10 @@ class LegacyBloomBitsBuilder : public BuiltinFilterBitsBuilder {
 
   Slice Finish(std::unique_ptr<const char[]>* buf) override;
 
-  int CalculateNumEntry(const uint32_t bytes) override;
-
-  uint32_t CalculateSpace(const int num_entry) override {
+  size_t CalculateSpace(size_t num_entries) override {
```

Comment:
Not sure if `const size_t` here would be better? What do you think?

Reply:
Google style prefers no const for the declaration:
> For a function parameter passed by value, const has no effect on the caller, thus is not recommended in function declarations.

For function definition / override, this is probably most applicable, since parameter definitions are local variables:
> Using const on local variables is neither encouraged nor discouraged.

My personal preference is to have the definition or implementations match the declaration. When I see 'const' in a function signature, I'm expecting a constraint on the caller, which doesn't apply here.